src/segments/sitecore.go

```diff
@@ -2,77 +2,96 @@ package segments
 
 import (
 	"encoding/json"
-	"path/filepath"
-	"strings"
+	"path"
 
 	"github.com/jandedobbeleer/oh-my-posh/src/platform"
 	"github.com/jandedobbeleer/oh-my-posh/src/properties"
 )
 
+const (
+	sitecoreFileName = "sitecore.json"
+	sitecoreFolderName = ".sitecore"
+	userFileName = "user.json"
+	defaultEnpointName = "default"
+)
+
 type Sitecore struct {
-	props properties.Properties
-	env   platform.Environment
+    props   	properties.Properties
+    env     	platform.Environment
+
+    EndpointName 	string
+	CmHost 			string
+}
 
-	Environment string
```

Comment:
This is a breaking change, why was this done?

Reply:
According to the Sitecore CLI terminology Environment is a complex object which contains cm-host, auth-host, ref, etc.
According to the current implementation, Environment has a value of cm-host.

The new implementation provides the same value but used for it a new property called CmHost and extends the Sitecore struct by providing a new property called EndpointName which contains the name of the environment configuration.

It was done to align with Sitecore CLI terminology.